### PR TITLE
Added the exit event

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

**Comment by favreau** [comment]
+1

**Comment by eile** [comment]
Removed in ZeroEQ/FlatBuffers?

**Comment by eile** [comment]
+1 otherwise

File: zerobuf/render/exit.fbs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori.Chevtchenko@epfl.ch
+//
+namespace zerobuf.render;
+
+table Exit
```
Comment (tribal-tec) [inline zerobuf/render/exit.fbs]: Comment when/how/why this event is/has to be sent?

**Comment by chevtche** [comment]
Removed in ZeroEQ/FlatBuffers? Ahmet is already doing that.

**Comment by chevtche** [comment]
Can this be merged

File: zerobuf/render/exit.fbs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori.Chevtchenko@epfl.ch
+//
+namespace zerobuf.render;
+
+//This event is used to gracefully close applications.
```
Comment (bilgili) [inline zerobuf/render/exit.fbs]: Add to the changelog

Comment (chevtche) [inline zerobuf/render/exit.fbs]: done

**Comment by chevtche** [comment]
Please merge

**Pull request merged**